### PR TITLE
Added new badges

### PR DESCRIPTION
**Pull request by Jonathan1600**
After open sourcing we can finally add some badges, two of them show as pending but I hope they will resolve soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,12 @@
   <img src="https://img.shields.io/npm/dt/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/minzip/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/min/@dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
```
Comment (ajhyndman) [inline README.md:9]: It looks like you've got this badge listed twice.  Were you trying to include something else, or should we just remove one?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,12 @@
...
+  <img src="https://img.shields.io/github/issues-raw/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/issues-closed-raw/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/license/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/commit-activity/m/dropbox/ttvc"/>
```
Comment (ajhyndman) [inline README.md:13]: It is neat that all of these are possible, but I feel some of them might be distracting from the interesting information, haha.  😅 

(Also, I'm not sure highlighting the very limited activity we've had so far is super helpful!  😉)

I think I'd rather remove these... for now.  Are you okay with that?

Comment (ajhyndman) [inline README.md:13]: It is neat that all of these are possible, but I feel some of them might be distracting from the interesting information, haha.  😅 

(Also, I'm not sure highlighting the very limited activity we've had so far is super helpful!  😉)

I think I'd rather remove these... for now.  Are you okay with that?

**Review by ajhyndman — CHANGES_REQUESTED** [review]

**Comment by Jonathan1600** [comment]
I agree those were just some options since I didn't know what else to include 😅

**Review by ajhyndman — APPROVED** [review]

**Comment by ajhyndman** [comment]
It's cool to see what's possible!

**Pull request merged**